### PR TITLE
Implement registration schema commands in LoginRadius CLI

### PR DESCRIPTION
**Pull request by RevanthTanneeru**
#### Please note that in case of below checklist not updated accordingly, the maintainers have the right to immediately close your pull request.

- [x] You created a branch from `develop` branch.
- [x] You PR is raised on `develop` branch and not on`main`.
- [x] You have read the [Contribution Guidlines](CONTRIBUTING.md) before creating this PR.

### Description of the Change

lr get schema --active, --all

lr add schema --fieldname

lr delete schema --feildname

I have added a new set of commands for LoginRadius CLI i.e registration schema commands. 

With the help of these we can manipulate and view registration schema through CLI

### CODE REVIEW

File: cmd/get/schema/schema.go [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+		},
+	}
+	fl := cmd.Flags()
+	fl.StringVarP(&all, "opt", "o", "", "options to get active fields or all fields")
+	return cmd
+}
+
+func get() error {
+	res, err1 := api.GetSites()
+	if res.Userlimit == 7000 {
```
Comment (mohammed786) [inline cmd/get/schema/schema.go:60]: What other things we are getting to check plan details?

File: cmd/get/schema/schema.go [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+		RunE: func(cmd *cobra.Command, args []string) error {
+			if all == "" {
+				return &cmdutil.FlagError{Err: errors.New("`opt` is required argument")}
+			}
+			return get()
+
+		},
+	}
+	fl := cmd.Flags()
+	fl.StringVarP(&all, "opt", "o", "", "options to get active fields or all fields")
```
Comment (mohammed786) [inline cmd/get/schema/schema.go:54]: as discussed can't we add two boolean flag like
```
lr get schema --all
lr get schema --active
```

**Review by mohammed786 — CHANGES_REQUESTED** [review]
PR looks good, just some minor changes needed, and just wanted to know what other details we are getting to fetch the user plan?

File: cmd/get/schema/schema.go (hunk 1 above)
Comment (RevanthTanneeru) [reply cmd/get/schema/schema.go:65]: We can also check the plan details with `Domainlimit` attribute from `SitesReponse` struct.

**Review by mohammed786 — APPROVED** [review]
Looks good

**Review by mohammed786 — APPROVED** [review]

**Pull request merged**